src/_scss/pages/modals/fullDownload/_filterBar.scss

```diff
@@ -0,0 +1,50 @@
+.download-filter-bar {
+    .search-top-filter-bar {
+        margin-bottom: rem(20);
+    }
+
+    .search-top-filter-header {
+        margin-bottom: rem(10);
+    }
+
+    .search-top-filters-content {
+        padding: rem(10);
+        background-color: $filter-bar-gray;
+
+        overflow-x: auto;
+        white-space: nowrap;
+        
+        .filter-group-container {
+            display: inline-block;
+
+            .filter-values {
+                white-space: nowrap;
+                .filter-item-container {
+                    display: inline-block;
+                    margin-left: rem(10);
+                }
+            }
+        }
+
+        .filter-item {
+            @include button-unstyled();
+            margin-top: 0.5rem;
+            margin-bottom: 0.5rem;
+            background-color: $color-white;
+            padding: 8px;
+            border: 1px solid $color-gray-light;
+            @include border-top-radius(4px);
+            @include border-bottom-radius(4px);
+
+            font-size: $smallest-font-size;
+
+            &:hover, &:active {
+                background-color: $color-white;
+            }
+
+            .filter-item-title {
+                color: $color-gray;
+            }
```

Comment:
Missing `.filter-item-remove-container` styling for the x icon
<img width="1165" alt="filter-item" src="https://user-images.githubusercontent.com/7108785/30760222-4ff0595e-9fa7-11e7-9d97-77601893ab99.png">

Reply:
This was an issue with the compressed version of the tags that should be displaying.  Contract filters (Type of Pricing,  Extent Competed, Set-Aside) were missing this boolean - updated!  Now no tags should have the option to remove in the download modal.